internal/tpl/makefile_test.go

```diff
@@ -249,4 +249,19 @@ makefile_version: 0.0.0
 		_, err := data.GetModuleDeps("m1")
 		Ω(err).Should(HaveOccurred())
 	})
+
+	It("IsNoSource fails on not existing module", func() {
```

Comment:
Did you mean to add a test where `IsNoSource` succeeds?

Reply:
no, I followed examples of GetModuleDeps that you provided